zipline/testing/core.py

```diff
@@ -853,22 +539,32 @@ def write_minute_data_for_asset(env, writer, start_dt, end_dt, sid,
         for minute in minute_blacklist:
             df.loc[minute] = 0
 
-    writer.write(sid, df)
+    return df
+
+
+def write_minute_data_for_asset(env, writer, start_dt, end_dt, sid,
+                                interval=1, start_val=1):
+    writer.writer(
```

Comment:
this is likely dead code because there is no `writer` method here. I will check and probably remove.